.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.js text eol=lf
+*.jsx text eol=lf
```

Comment:
Can these files have new lines added to end please?

Reply:
I think Drew & I agreed we're turning this into a separate issue, along with a Prettier rule. 😄